src/styles/blocks/content.scss

```
@@ -0,0 +1,121 @@
+.content {
+  &_row {
+    display: flex;
+    justify-content: space-around;
+    max-width: ($card-width * 4) + (($card-gap * 3) * 4);
+    margin: 0 auto;
+    margin-top: 50px;
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  height: 406px;
+  padding: ($card-gap * 2) $card-gap $card-gap;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  align-items: center;
+  justify-content: center;
+  font-weight: normal;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__tittle {
+    margin: 0 0 4px;
+    color: var($color-amount);
+
+    @include card-text-param(500, 12px, $line-height);
+  }
+
+  &__card-id {
+    @include card-text-param(normal, 10px, 14px);
+
+    align-self: start;
+    color: $color-text-grey;
+  }
+
+  &__reit {
+    @extend %flex-space-between;
+
+    align-items: flex-end;
+    margin-top: $card-gap;
+    width: 100%;
+  }
+
+  &__rewiews {
+    @include card-text-param(normal, 10px, 14px);
+
+    color: $color-amount;
+  }
+
+  &__price {
+    @extend %flex-space-between;
+    height: $line-height;
+    line-height: $line-height;
+    align-items: center;
+    width: 100%;
+    margin-top: $card-gap + ($card-gap / 2);
+
+    &-name {
+      font-size: 12px;
+      line-height: $line-height;
+      color: $color-text-grey;
+    }
+
+    &-amount {
+      @include card-text-param(bold, 16px, $line-height);
+
+      color: $color-amount;
+    }
+  }
+
+  .card__buy {
```

```suggestion
  &__buy {
```
Чому б не перенести цей елемент в блок card, як ти робиш з попередніми класами?